### DISCUSSION TITLE
Issue with vendore (repo's language stats absent)

### DISCUSSION THREAD

**Discussion by michalspano** [github-linguist/linguist#5548] [Classification] [answered]
Hello!
I am aware that this issue has been addressed several times, however, I don't seem to find any useful solution, I thus seek help via opening a __Discussion's thread__.

I own a __public repositery__ ([link][LINK1]) and the __src__ is currently under `docs/` folder (which happens to be _vendored_ by __Linguist's__ default standards at [`languages.yml`][LINK2]).

I tried adding [`.gitattributes`][LINK3] to override default behaviour with such code:

```
docs/** linguist-vendored=false
```

Unfortunately, it still seems like that the __language stats__ are not included (from `docs/`).

Any guidance will be highly appreciated. 😊 

[LINK1]: https://github.com/michalspano/The-Swedish-Bot
[LINK2]: https://github.com/github/linguist/blob/master/lib/linguist/languages.yml
[LINK3]: https://github.com/michalspano/The-Swedish-Bot/blob/main/.gitattributes

**Comment by lildude** [comment, marked as answer]
> (which happens to be _vendored_ by **Linguist's** default

This is your issue... `docs/` is considered documentation, not vendored:

https://github.com/github/linguist/blob/0fca6f46491731d9adedc678d8663aa74d40a5a6/lib/linguist/documentation.yml#L10-L12

Accordingly, you need to use the `-linguist-documentation` or `linguist-documentation=false` override.

**Comment by michalspano** [comment]
Thanks, @lildude.
I apologize, I might have misread the __docs__ at this point.

Hopefully this clears it up and be helpful for other users! 😊
